### DISCUSSION TITLE
Nô lệ của đội tinh nhuệ Ma đô - SuicaoDex

### DISCUSSION THREAD

**Discussion by giscus** [TNTKien/suicaodex#7] [Manga]
# Nô lệ của đội tinh nhuệ Ma đô - SuicaoDex

Chiều không gian đầy quỷ dữ được gọi là "Ma đô" đã mở ra đe dọa đối với loài người. Tại đó, có một loài  "đào" ban cho kẻ ăn sức mạnh đặc biệt, giúp nhân loại có cơ hội chiến đấu, tuy nhiên, chỉ có nữ giới mới có thể sử dụng thứ sức mạnh "trái đào" ấy. Bởi vậy mà phụ nữ trở thành những người bảo vệ loài người, địa vị xã hội tăng cao, trong khi nam giới bị giáng xuống tầng lớp công dân hạng hai, giành giật nhau những công việc cơ bản và sự thừa nhận.

Nhân vật chính Yuuki Wakura, một nam sinh chăm chỉ yêu nội trợ, chị gái cậu đã bị bắt trong một sự kiện của Ma đô 5 năm về trước. Khi đang than thở về những khó khăn gặp phải, cậu đã bị lũ quỷ tấn công. Nhờ được đội trưởng Kyouka của Đội chống Quỷ kịp thời ứng cứu, cậu chấp nhận trở thành "nô lệ" của cô để có thể sở hữu sức mạnh tiêu diệt bầy quỷ.

Bất ngờ thay, việc làm "nô lệ" đòi hỏi một cái giá tương xứng...và người phải trả giá lại không phải là cậu.

https://suicaodex.com/manga/e1e38166-20e4-4468-9370-187f985c550e

**Comment by TNTKien** [comment]
Heh

**Reply by suicaodeptrai** [reply]
![scarseven](https://github.com/user-attachments/assets/5f886711-5537-4edc-9c05-f6fe9194ce89)

**Comment by suicaodeptrai** [comment]
Chap 146 delay qua tuần sau nhé ae ![kyaru](https://cdn.discordapp.com/emojis/1236011754631463013.gif)

**Comment by ngocanhsv96** [comment]
Cái đoạn 2 đứa kia trao thưởng gần cuối là ở chap nào thế ad?

**Reply by iamneyk** [reply]
là [chap 84](https://suicaodex.com/chapter/77890bc7-764c-4f44-a67e-3034714c15e2) và [chap 90.5](https://suicaodex.com/chapter/687c540f-fbf3-49f6-a22c-1d5a03d7aa3d)

**Comment by 0xTOHO** [comment]
# **Cái này bình luận được trên app github # #  # # mobie*
@blogtruyenchu

**Reply by TNTKien** [reply]
chắc chắn rồi, cmt ngay trên discussion cũng sẽ hiện trong truyện

**Comment by neluon113** [comment]
qua tuyet voi hong anime khi nao ss2 vau mn
